src/prefect/_internal/schemas/validators.py

```diff
@@ -375,8 +375,6 @@ def reconcile_paused_deployment(values):
 
 
 def default_anchor_date(v: DateTime) -> DateTime:
-    if v is None:
-        return pendulum.now("UTC")
     return pendulum.instance(v)
```

Comment:
I _think_ if we're using the `pydantic_extra_types.pendulum` stuff, we will always get things as pendulum instances?

Reply:
Possibly I can play with that and make sure.

Reply:
It's unhappy about having stdlib datetime passed in without it.

Reply:
I think the model has to declare the type as `DateTimeTZ` (which is just our alias for the `pydantic_extra_types` one)

Reply:
It's declaring it as the `DateTime` directly from `pydantic_extra_types`.